install.sh

```diff
@@ -994,11 +994,27 @@ case "${SHELL}" in
     ;;
 esac
 
-cat <<EOS
+in_shell_profile=$(grep -qs "eval \"\$(${HOMEBREW_PREFIX}/bin/brew shellenv)\"" "${shell_profile}")
+
+# FIXME: figure out why this give a "command not found" error
+# show different instructions on adding Homebrew to path based on if the user has already done so
+if "${in_shell_profile}" && ! type brew &>/dev/null
+then
+  cat <<EOS
+- Run this command in your terminal to add Homebrew to your ${tty_bold}PATH${tty_reset}:
+    eval "\$(${HOMEBREW_PREFIX}/bin/brew shellenv)"
+EOS
+elif ! "${in_shell_profile}"
```

Comment:
If you use two `if`s here you can avoid checking the same condition multiple times, no?

Reply:
Yeah, I could use a nested if. I will do so.

Reply:
By using the nested `if` you can again make it a direct command (`if grep`) instead of saving to a variable, which should handle the error you got above.

Reply:
I see. That would make more sense to do now since we are only comparing it once.